cmd/podman/system_df.go

```diff
@@ -460,11 +460,11 @@ func getImageVerboseDiskUsage(ctx context.Context, images []*image.Image, images
 		}
 		var repo string
 		var tag string
-		if len(img.Names()) == 0 {
-			repo = "<none>"
-			tag = "<none>"
+		var repotags []string
+		if len(img.Names()) != 0 {
+			repotags = []string{img.Names()[0]}
 		}
-		repopairs, err := image.ReposToMap([]string{img.Names()[0]})
+		repopairs, err := image.ReposToMap(repotags)
```

Comment:
If len(img.Names()) == 0, shouldn't we still be filing up repo and tag with "<none>"?

Reply:
No. image.ReposToMap can handle the empty slice as the input argument and return the map with key:value pairs ("<none>","<none>")

Reply:
OK, if you're sure it's happy.  I've  a vague memory of a that "<none>" code being added to address another problem.

Reply:
ugh love md autoformat sometimes "...memory of that `<none>` code..."